user_docs/en/changes.t2t

```diff
@@ -7,6 +7,10 @@ What's New in NVDA
 = 2024.2 =
 
 == New Features ==
+- A new braille mode called "display speech output" has been added. (#15898, @Emil-18)
+  - When active, the braille display shows what NVDA is speaking.
+  - It can be toggled by pressing NVDA+alt+t, or from the braille settings dialog.

```

Comment:
Fix up

```suggestion
- A new braille mode called "display speech output" has been added. (#15898, @Emil-18)
  - When active, the braille display shows exactly what NVDA is speaking.
  - It can be toggled by pressing ``NVDA+alt+t``, or from the braille settings dialog.
  -
```